client/src/pages/auth/formProps.js

```diff
@@ -0,0 +1,59 @@
+// TODO: Somewhat overloaded, probably should be
+// restructured if it needs to scale in the future

```

Comment:
Yeah probably. Okay for now though

Reply:
I also want to just note for everyone that different levels of modularity depend on preferences. This is very modular, but some would prefer having these props directly on the component because it's:
1. more transparent (you don't have to go to another file to check the props
2. more customizable (what if you want to have a field with just one different prop. Easier to see and change on the component itself)
3. avoids the problem of having a giant file with lots of slightly different props. Similar to how the trend is to move towards makeStyles on the page instead of a giant stylesheet.